USING_DOCKER.md

```diff
@@ -0,0 +1,93 @@
+# Using the Dockerfile
+
+The `Dockerfile` in this repo builds a Docker container
+that has current versions of all the development tools.
+Using Docker means you do not have to concern
+yourself that these tools will conflict with
+other software on your computer.
+
+First install Docker on your computer,
+using one of the many guides on the internet.
+
+Next build the Docker container with the following command.
+_(It will take some time while Docker pulls in all the
+necessary developer tools.)_
+
+```
+cd <top-level-PRQL-directory>
+docker build -t prql .
+```
+
+Finally, start up the Docker container with:
+
+```
+cd <top-level-PRQL-directory>
+docker run -it -v $(pwd)/:/app -p 3000:3000 prql
+```
+
+- You'll see a `root@xxxxxxxxx:/app/#` prompt
+- Enter the commands below for the component you're working on
+- Ctl-C to exit that component
+- Enter 'exit' to close down the Docker machine
+
+## Running components under Docker
+
+The following commands come from the `README.md`
+file in each of the component directories.
+Use the `docker run...` command above, then:
+
+**Playground** Use the command above, then enter:
+
+```
+cd playground; npm start
+```
+
+**Language Book** Use the command above, then enter:
+
+```
+cd book; mdbook serve -n 0.0.0.0 -p 3000
```

Comment:
Nice. A potential follow-up would be to set the default port in `mdbook` confighs so we don't even need to supply it here (and same for hugo)